package.json

```diff
@@ -6,10 +6,12 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "final-form": "^4.20.4",
```

Comment:
`final-form` and `react-final-form` don't appear to be used - can they be removed?